sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -85,17 +85,16 @@ export interface ServiceBusSender {
    */
   createMessageBatch(options?: CreateMessageBatchOptions): Promise<ServiceBusMessageBatch>;
 
-  // TODO: Commented out to come up with an alternative name
-  // /**
-  //  * Opens the AMQP link to Azure Service Bus from the sender.
-  //  *
-  //  * It is not necessary to call this method in order to use the sender. It is
-  //  * recommended to call this before your first sendMessages() call if you
-  //  * want to front load the work of setting up the AMQP link to the service.
-  //  *
-  //  * @param options - Options to configure tracing and the abortSignal.
-  //  */
-  // open(options?: OperationOptionsBase): Promise<void>;
+  /**
+   * Pre-establish the AMQP link to Azure Service Bus from the sender.
+   *
+   * It is not necessary to call this method in order to use the sender. It is
+   * recommended to call this before your first sendMessages() call if you
+   * want to front load the work of setting up the AMQP link to the service.
```

Comment:
nitpick: rewording suggestion, feel free to edit it though:
```suggestion
   * Establishes the AMQP link to Azure Service Bus.
   *
   * Call this method before your first `sendMessages()` call if the overhead of link setup is not tolerated at sending time. Otherwise, it is not required to call this method explicitly.
```